dev_guide/jobs.adoc

```diff
@@ -76,9 +78,10 @@ spec:
 A job can be scaled up or down by changing the `*parallelism*` parameter
```

Comment:
@kargakis I think I see what you mean. Maybe something like:

"A job can be scaled up or down by manually changing the `*parallelism*` parameter of a configuration file accordingly, or by using the `--replicas` option with the `oc scale` command, which, in the case of a job, modifies the `*parallelism*` parameter."

It's a big sentence, though. @soltysh It might be best to pick one method to suggest to the reader, and that can be the official way to scale jobs.